compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CompilerPluginRunner.java

```diff
@@ -301,7 +301,7 @@ private void handleServiceTypeProcesses(CompilerPlugin plugin) {
             BPackageSymbol symbol = this.packageCache.getSymbol(packageQName);
             if (symbol != null) {
                 SymbolEnv pkgEnv = symTable.pkgEnvMap.get(symbol);
-                final BSymbol listenerSymbol = symResolver.lookupSymbol(pkgEnv, listenerName, SymTag.OBJECT);
+                final BSymbol listenerSymbol = symResolver.lookupMainSpaceSymbol(pkgEnv, listenerName);
```

Comment:
Need to check for Object symbol after this.

Reply:
Changed